header/partials/header/top.html

```diff
@@ -26,7 +26,7 @@
 							<span class="o-header__visually-hidden">myFT</span>
 						</a>
 						{{#if @root.flags.myftHint}}
-							{{#unless @root.locals.anon.userIsAnonymous}}
+							{{#unless @root.anon.userIsAnonymous}}
```

Comment:
This doesn’t look quite right… The anon object is [still in locals, no](https://github.com/Financial-Times/n-express/blob/master/src/anon/middleware.js#L11)?